types/artifacts.d.ts

```diff
@@ -64,8 +64,8 @@ declare global {
      * on a major version bump.
      */
     export interface PublicGathererArtifacts {
-      /** Console deprecation and intervention warnings logged by Chrome during page load. */
-      ConsoleMessages: Crdp.Log.EntryAddedEvent[];
+      /** Console deprecation and intervention warnings, Console API calls, and exceptions logged by Chrome during page load. */
+      Console: Artifacts.ConsoleMessage[];
```

Comment:
bike shedding (sorry!): to me, `ConsoleMessages` probably still makes the most sense for this artifact name. Plural indicating that it's a collection of `ConsoleMessage` instances, and it matches the pattern of a lot of the other public artifacts (`IframeElements`, `ScriptElements`, `ViewportDimensions`, etc)

Reply:
SG